src/media/js/addon/components/search.js

```diff
@@ -37,12 +37,14 @@ export default class Search extends React.Component {
 
         {this.props.results.length && this.state.q &&
           <ul className="search-results">
-            {this.props.results.map(result =>
-              <li className="search-result">
-                <ReverseLink to="addon-review-detail"
-                             params={{slug: result.slug}}>
-                  {result.name} ({result.slug})
-                </ReverseLink>
+            {this.props.results.map((result, i) =>
```

Comment:
This is really painful to read. Can you move the function into a render method so you can oneline it?